app/controllers/projects_controller.rb

```diff
@@ -3,6 +3,8 @@ class ProjectsController < ApplicationController
   after_action :verify_authorized
   after_action :verify_policy_scoped, only: :index
 
+  DEFAULT_WORKFLOW_ROOT = Rails.root.join("app", "workflows", "lists", "conversion").freeze
```

Comment:
Only question I have is whether this should live here or in the `TaskListCreator`.

I suppose you could do this, which would allow you to pass arguments in specs when needed, but also keeps the path in the `TaskListCreator`

```ruby
class TaskListCreator
  DEFAULT_WORKFLOW_ROOT = Rails.root.join("app", "workflows", "lists", "conversion").freeze

  def call(project, workflow_root: DEFAULT_WORKFLOW_ROOT)
     workflow = load_workflow_definition(workflow_root)
     create_task_list_from_workflow(project, workflow)
   end
```